docs/getting-started-guides/windows/index.md

```diff
@@ -17,28 +17,7 @@ The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) co
 ## Build
 We recommend using the release binaries that can be found at [https://github.com/kubernetes/kubernetes/releases](https://github.com/kubernetes/kubernetes/releases). Look for the Node Binaries section by visiting the binary downloads link. 
 
-If you wish to build the code yourself, please follow the next instructions:
-
-1. Install the pre-requisites on a Linux host:
-```
-sudo apt-get install curl git build-essential docker.io conntrack
-```
-2. Run the following commands to build kubelet and kube-proxy:
-```
-K8SREPO="github.com/kubernetes/kubernetes"
-go get -d $K8SREPO
-# Note: the above command may spit out a message about 
-#       "no Go files in...", but it can be safely ignored!
-
-cd $GOPATH/src/k8s.io/kubernetes
-# Build the kubelet
-KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kubelet
-
-# Build the kube-proxy
-KUBE_BUILD_PLATFORMS=windows/amd64 make WHAT=cmd/kube-proxy
-
-# You will find the output binaries under the folder _output/local/bin/windows/
-```
+If you wish to build the code yourself, please follow the [Compiling Kubernetes Binaries instructions](https://github.com/microsoft/Virtualization-Documentation/blob/live/virtualization/windowscontainers/kubernetes/compiling-kubernetes-binaries.md)
```

Comment:
Any reason for this? I thought we discussed it here: https://github.com/JMesser81/kubernetes.github.io/pull/1#discussion_r154569768
I agree with Michael on this. I also prefer to have the steps on how to build K8s for windows in this page.

Reply:
Sure. I'll place the build instructions back on this main index page